OpenEdXMobile/src/main/java/org/edx/mobile/whatsnew/WhatsNewModel.java

```diff
@@ -7,6 +7,9 @@
 import java.util.List;
 
 public class WhatsNewModel implements Parcelable {
```

Comment:
`WhatsNewModel.java` should be the name of current `WhatsNew.java` class. This class should be named `WhatsNewVersionModel.java` or `WhatsNewItemModel.java`